doc/content/getting-started/migrating-from-v2/export-v2-devices.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Export End Devices From V2
+weight: 40
+---
+
+## Export End Devices with ttnctl
+
+In this step, the end devices from {{% ttnv2 %}} will be exported in a JSON format that can then be parsed and imported by {{% tts %}}.
+
+For the commands below, an application on {{% ttnv2 %}} is assumed, with **AppID** `v2-application` and **AppEUI** `0011223300112233`.
+
+Select that **AppID** and **AppEUI** with:
+
+```bash
+$ ttnctl applications select
+```
+
+After selecting the application, make sure that you can list the available end devices:
+
+```bash
+$ ttnctl devices list
+```
+
+In order to export a single device, use the following command. The device will be saved to `device.json`.
+
+```bash
+$ ttnctl devices export "DeviceID" --frequency-plan-id EU_863_870 > device.json
+```
+
+Alternatively, you can export all the end devices with a single command and save them in `all-devices.json`.
+
+```bash
+$ ttnctl devices export-all --frequency-plan-id EU_863_870 > all-devices.json
+```
+
+> **NOTE**: You can find a list of supported frequency plan IDs in [the lorawan-frequency-plans Github repository](https://github.com/TheThingsNetwork/lorawan-frequency-plans/blob/master/frequency-plans.yml).
```

Comment:
This should be a reference

See #2421

Reply:
I thought by 'no hugo magic' that you meant let's just continue to link to github. we DO want to manually copy the frequency plans to the reference section?

Reply:
Leaving the link to the GitHub repo for now, with a TODO comment to link to our documentation page after #2421 has been merged.

Reply:
OK I'm fine with a TODO comment as long as there is a (specific) open issue that's referenced